import.go

```diff
@@ -91,6 +91,13 @@ func importBuilderFromImage(store storage.Store, options ImportFromImageOptions)
 	}
 	img, err := is.Transport.GetStoreImage(store, ref)
 	if err != nil {
+		img2, err2 := store.Image(options.Image)
```

Comment:
Yeah, that'd work, too.  I thought one of the linters would complain about returning from one leg of an if-then-else (since you can rewrite it as if-then-return, and just have the else bit be the continuation), but it didn't.